src/Equinor.ProCoSys.IPO.Query/GetInvitationById/InvitationDto.cs

```diff
@@ -16,6 +16,7 @@ public class InvitationDto
             PersonDto createdBy,
             DateTime startTimeUtc,
             DateTime endTimeUtc,
+            bool canEdit,
```

Comment:
Consider renaming to editable.

Reply:
I will leave it like this, since we have used this wording in canSign feks